src/tests/functional/plugin/shared/src/behavior/compiled_model/import_export.cpp

```diff
@@ -298,6 +298,24 @@ TEST_P(OVClassCompiledModelImportExportTestP, smoke_ImportNetworkNoThrowWithDevi
     OV_ASSERT_NO_THROW(executableNetwork.create_infer_request());
 }
 
+TEST_P(OVClassCompiledModelImportExportTestP, smoke_ImportNetworkThrowWithDeviceName) {
+    // only CPU plugin supports this exception now.
+    if (target_device != "CPU") {
```

Comment:
please, don't use conditions in tests. We have skip config for it like https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

Reply:
Sure, thanks!